easybuild/easyconfigs/l/LinBox/LinBox-1.4.0-foss-2016a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild recipe; see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright (c) 2016 Riccardo Murri <riccardo.murri@gmail.com>
+# Authors::   Riccardo Murri <riccardo.murri@gmail.com>
+# License::   GPL
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'LinBox'
+version = '1.4.0'
+
+homepage = 'http://linalg.org/'
+description = "C++ library for exact, high-performance linear algebra"
+
+toolchain = {'version': '2016a', 'name': 'foss'}
+
+sources = ['v1.4.0.zip']
+source_urls = ['https://github.com/%(namelower)s-team/%(namelower)s/archive']
+
+builddependencies = [
+    ('Autoconf', '2.69', '', True),
+    ('Automake', '1.15', '', True),
+    ('libtool', '2.4.6', '', True),
+]
+dependencies = [
+    ('FFLAS-FFPACK', '2.2.0'),
+    ('Givaro', '4.0.1'),
+]
+
+preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
+configopts = "--with-givaro=$EBROOTGIVARO --with-fflas-ffpack=$EBROOTFFLASMINFFPACK --enable-openmp"
+
+parallel = 4
+
+unwanted_env_vars = {}
+start_dir = '%(builddir)s/%(namelower)s-%(version)s/'
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s-config', 'include/%(namelower)s/%(namelower)s-config.h', 'lib/liblinbox.a'],
```

Comment:
either always use `%(namelower)s`, or always use `linbox`, don't mix (I prefer the latter, but both are fine)